From 4eb1a3da7d65ae798db324153d693adef13ef6ed Mon Sep 17 00:00:00 2001
From: Bui Quang Minh <minhquangbui99@gmail.com>
Date: Mon, 4 Oct 2021 20:38:34 +0700
Subject: [PATCH 010/120] criu(8): Add more detailed description about
 --tcp-close dump option

The expected behavior of --tcp-close option when dumpping is to close
all established tcp connections including connection that is once
established but now closed. This adds an explicit description about
that behavior.

Signed-off-by: Bui Quang Minh <minhquangbui99@gmail.com>
---
 Documentation/criu.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/criu.txt b/Documentation/criu.txt
index 3c4c1eaf8..4c6885fc9 100644
--- a/Documentation/criu.txt
+++ b/Documentation/criu.txt
@@ -334,7 +334,8 @@ mount -t cgroup -o devices,freezer none devices,freezer
     Checkpoint established TCP connections.
 
 *--tcp-close*::
-    Don't dump the state of, or block, established tcp connections.
+    Don't dump the state of, or block, established tcp connections
+    (including the connection is once established but now closed).
     This is useful when tcp connections are not going to be restored.
 
 *--skip-in-flight*::
-- 
2.34.1

